src/loaders/express.loader.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { usersRouter } from '../api';
+import { config } from '../config';
+
+export default async ({ app }: { app: express.Application }) => {
```

Comment:
why do you need async here? You don't have any awaiting for asynchronous code

Reply:
done